.github/workflows/continuous.yml

```diff
@@ -90,19 +94,22 @@ jobs:
           cmake -G Ninja ^
             -DIPC_TOOLKIT_BUILD_TESTS=ON ^
             -DCMAKE_BUILD_TYPE=${{ matrix.config }} ^
+            -DCMAKE_CXX_FLAGS=" /DWIN32 /D_DISABLE_CONSTEXPR_MUTEX_CONSTRUCTOR" ^
```

Comment:
This is a temporary fix based on https://github.com/actions/runner-images/issues/10004